components/Nav.js

```diff
@@ -148,6 +146,30 @@ const navItems = [
 
 
 class Nav extends Component {
+  /***************************************************************************\
+    Properties
+  \***************************************************************************/
+
+  state = {
+    openSubNav: '',
+  }
+
+
+  /***************************************************************************\
+    Private Methods
+  \***************************************************************************/
+
+  _handleSubnavChange = ({ target }) => {
+    const { openSubNav } = this.state
+    const { id } = target
+
+    this.setState({ openSubNav: openSubNav === id ? '' : id })
```

Comment:
`_handleSubnavChange` vs `this.state.openSubNav`... Consistency, man!

Reply:
whoops XD